src/models/searchservice/response/directanswer/StringValueDirectAnswer.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseFieldValueDirectAnswer } from './BaseFieldValueDirectAnswer';
+import { BuiltInFieldType } from './BuiltInFieldType';
+
+/**
+ * A direct answer whose source is a string or string list field in the knowledge graph.
+ *
+ * @public
+ */
+export interface StringValueDirectAnswer extends BaseFieldValueDirectAnswer<string | string[]> {
+  fieldType: BuiltInFieldType.SingleLineText | BuiltInFieldType.MultiLineText
```

Comment:
Would single line text ever have a string[] value? For my edification what search queries did you use? (so I can store them in my brain for future use)

Reply:
You can create a custom field that is single line text [], which translates to string[]. I just searched "Abington Services Offered". Services Offered was the single line text[] field.